core/src/main/java/org/openstack4j/openstack/storage/block/domain/CinderVolumeAttachment.java

```diff
@@ -14,60 +14,61 @@
 @JsonRootName("volumeAttachment")
 public class CinderVolumeAttachment implements VolumeAttachment {
 
-	private static final long serialVersionUID = 1L;
-
-	@JsonProperty
-	private String device;
-
-	@JsonProperty
-	private String host_name;
-
-	@JsonProperty
-	private String id;
-
-	@JsonProperty
-	private String server_id;
-	
-	@JsonProperty
-	private String attachment_id;
-
-	@JsonProperty
-	private String volume_id;
-
-	@Override
-	public String getDevice() {
-		return device;
-	}
-
-	@Override
-	public String getHostname() {
-		return host_name;
-	}
-
-	@Override
-	public String getId() {
-		return id;
-	}
-
-	@Override
-	public String getServerId() {
-		return server_id;
-
-	}
-
-	@Override
-	public String getVolumeId() {
-		return volume_id;
-	}
-
-	@Override
-	public String toString() {
-		return MoreObjects.toStringHelper(this).omitNullValues()
-				.add("device", device).add("id", id).add("serverId", server_id)
-				.add("volumeId", volume_id).add("hostname", host_name).toString();
-
-	}
-
-
-
+    private static final long serialVersionUID = 1L;
```

Comment:
spaces or tabs? @olivergondza 
some files use spaces, others tabs

Reply:
Yeah, current code style is a mess. Lest just stick with whatever the particular file is using for now (provided it is consistent internally) in order to keep the diffs minimal.

I have no experience with [.editorconfig](https://github.com/openstack4j/openstack4j/issues/21) you have suggested, so that is something to have a look at. If not for the sake of consistency, then to get rid of that blasted tabs.